djangoevents/schema.py

```diff
@@ -8,13 +8,23 @@
 import stringcase
 
 from avro.io import Validate as avro_validate
+from eventsourcing.domain.model.events import DomainEvent
```

Comment:
Can we use the internal import for the `DomainEvent` ? (https://github.com/ApplauseOSS/djangoevents/blob/devel/djangoevents/__init__.py#L22)